### DISCUSSION TITLE
Client session cache is calculated in accordance with realm setting but advanced client settings are ignored

### DISCUSSION THREAD

**Discussion by Spiritwisp** [keycloak/keycloak#20458] [Q&A]
Found that clientSessions cache was calculated in accordance with realm setting but advanced client settings are ignored.
[here](https://github.com/keycloak/keycloak/blob/34a68d935497af489ceec5e6c541995092147991/model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/util/SessionTimeouts.java#LL54C20-L54C20)
the clientSessions cache takes only realm setting
![image](https://github.com/keycloak/keycloak/assets/57109587/6bac1554-db4b-4078-b618-6e9fe81ebe04)
but advanced settings are ignored
![image](https://github.com/keycloak/keycloak/assets/57109587/1cc2b760-ca0e-41df-9fef-837a3d393922)

**Comment by jonkoops** [comment]
If this is a bug please report it as such, and do provide more information to reproduce it. Looking at the screenshots you have posted this is an older version of Keycloak, so make sure to upgrade to the latest version and ensure this is still an issue before reporting.
